src/ConnectionPool.jl

```diff
@@ -410,7 +410,7 @@ end
 function getconnection(::Type{TCPSocket},
                        host::AbstractString,
                        port::AbstractString;
-                       keepalive::Bool=false,
+                       keepalive::Bool=true,
```

Comment:
```suggestion
                       # set keepalive to true by default since it's cheap and helps keep long-running requests/responses
                       # alive in the face of heavy workloads where Julia's task scheduler might take a while to
                       # keep up with midflight requests
                       keepalive::Bool=true,
```